src/client/utils/dom.js

```diff
@@ -16,6 +16,20 @@ const arraySlice = Array.prototype.slice;
 
 var scrollbarSize = null;
 
+const ELEMENT_TEXT_PROPERTIES = ['innerHTML', 'outerHTML', 'innerText', 'textContent', 'text'];
+
+function getAttrValues (element) {
+    var values = {};
+
+    for (var property of ELEMENT_TEXT_PROPERTIES)
+        values[property] = element[property];
+
+    return values;
+}
+
+const DEFAULT_SVG_ELEMENT_TEXT_ATTR_VALUES  = getAttrValues(nativeMethods.createElementNS.call(document, 'http://www.w3.org/2000/svg', 'svg'));
```

Comment:
`SVG_ELEMENT_TEXT_PROPERTIES`

Now it contains empty string or undefined values. 
Instead of this, you need to store property existence.